playground/config/demo-virtualService.yaml

```
@@ -14,6 +14,6 @@ spec:
         prefix: /
     route:
     - destination:
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)